### ISSUE TITLE
Albums view very slow

### ISSUE THREAD

**Issue by gs11** [Sonerezh/sonerezh#279]
My collection contains 767 albums (10014 songs).
When browsing by album the first load is very slow causing MySQL to use 100% CPU and finally causing a timeout in the browser.
Subsequent browses are quicker but scrolling to the bottom to display the next n albums results in the same behavior but no timeout since it's quicker.

Running sonerezh (git master) on a nginx/debian machine with 2.4 GHz CPU / 4GB RAM.

**Comment by gs11** [comment]
Ok, the mysql query is the culprit:
```sql
 SELECT `Song`.`id`, `Song`.`band`, `Song`.`album`, `Song`.`cover` FROM `sonerezh`.`songs` AS `Song` WHERE (`Song`.`id`, `Song`.`album`) IN (SELECT MIN(`subsong`.`id`), `subsong`.`album` FROM `sonerezh`.`songs` AS `subsong`   WHERE 1 = 1  GROUP BY `subsong`.`album`)  ORDER BY `Song`.`album` ASC  LIMIT 216, 36
```

Why is a sub query (slow) used when the following statement yields the same result?
```sql
SELECT MIN(id), band, album, cover FROM songs GROUP BY album LIMIT 216, 36;
```

**Comment by gs11** [comment]
Newly created pull request #282 fixes this.

**Comment by davebiffuk** [comment]
With this change, the album view for my collection (~700 albums, ~7000 mp3 files) loads in just a second or so, hosted on a tiny ARM-based server - thanks!

**Comment by gs11** [comment]
Fix in PR #339
